### DISCUSSION TITLE
bug: Json to struct 对 时间类型的匹配不准确

### DISCUSSION THREAD

**Discussion by alctny** [he3-app/he3-feedback#156] [Bug Report]
![json-to-struct类型推断不准确](https://user-images.githubusercontent.com/44720162/233817605-da6f983a-e2ed-4a37-9245-ba433773ad79.png)

**Comment by lyzhang1999** [comment]
感谢你的反馈，我已经记录了该问题，并建立了 issues 来跟踪：https://github.com/he3-app/he3-feedback/issues/157 ， 您可以关注该 issues 以获得修复进度。
